LondonBritishLibrary/orient/BLorient812.xml

```diff
@@ -30,17 +31,99 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 384</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
                         <summary/>
-                        
+                        <msItem xml:id="ms_i1">
+                          <locus from="3r"></locus>
+                          <title type="incomplete" ref="LIT1011Abusak"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <note>Chapters 47, 49, 51, 53, 58 and 59 are missing.</note>
+
+                          <msItem xml:id="ms_i1.1">
+                            <locus from="3r"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Index"></title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.2">
+                            <locus from="5v"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Chapter1"></title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.3">
+                            <locus from="97r"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Chapter48"></title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.4">
+                            <locus from="154r"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Chapter50"></title>
+                            <textLang mainLang="gez"></textLang>
+
+                            <msItem xml:id="ms_i1.4.1">
+                              <title type="complete" xml:lang="en">The list of the patriarchs
+                              of Alexandria</title>
+                              <explicit xml:lang="gez"><locus target="180v"></locus>
+                              ፺ወ፯ <persName ref="PRS4381Gabriel">አባ፡ ገብርኤል፡</persName> ዘሀሎ፡ በዘመነ፡
+                              <persName ref="PRS8550sardaDe">ንጉሥ፡ ሠርፀ፡ ድንግል።</persName>
+                            <sic>፺ወ፰</sic> <persName ref="PRS5710JohnXV">አባ፡ ዮሐንስ።</persName>
+                            <sic>፺ወ፱</sic> <persName ref="PRS6905Matthew">አባ፡ ማቴዎስ።</persName>
+                            <sic>፻</sic> <persName ref="PRS6752Mark">አባ፡ ማርቆስ።</persName>
+                              <!-- the numbers are not conform to the traditional system, Mark V is
```

Comment:
I think that his could be changed into a note: if you noticed this, others also might find it interesting.